wremnants/combine_theory_helper.py

```diff
@@ -564,9 +564,7 @@ def add_pdf_uncertainty(self, operation=None, scale=-1.):
         pdf_ax = self.syst_ax if self.pdf_from_corr else "pdfVar"
         symHessian = pdfInfo["combine"] == "symHessian"
 
-        processesZ = [] if self.skipFromSignal else ['single_v_samples']
-        processesW = ['wtau_samples', 'single_v_nonsig_samples'] if self.skipFromSignal else ['single_v_samples']
-        processes = processesW if self.label=="W" else processesZ
+        processes = ['single_v_samples']
```

Comment:
Also here, this seems to remove some functionality. Is this change needed?

Reply:
Same as up. But for PDFs

Reply:
But if skipFromSignal is false you should get the same as before. It's just that the new implementation removes the skipFromSignal functionality, maybe this was needed for some tests or so. Maybe @kdlong know?

Reply:
Yes I agree that this change can simply be reverted and the helicity fit should run fine, since as David said unless you use skipFromSignal for some reason there is no actual change here.

Reply:
Ok so after discussing with @emanca I understand better that the skipFromSignal option exists precisely for the helicity fit, and the change in behaviour here is intentional (to add back these uncertainties even when the option is enabled)

So this can also stay as it is.